apps/remix-ide/src/app/files/fileProvider.js

```diff
@@ -35,7 +36,12 @@ class FileProvider {
   }
 
   getPathFromUrl (url) {
-    return this.providerExternalsStorage.get('reverse-' + url)
+    return this.providerExternalsStorage.get(this.reverseKey + url)
+  }
+
+  getUrlFromPath (path) {
```

Comment:
should remixdProvider also have an implementation for this?

Reply:
it already has, remixdProvider has fileProvider has base class.